### DISCUSSION TITLE
hide moving average priceLine

### DISCUSSION THREAD

**Discussion by itsDantes** [tradingview/lightweight-charts#1124] [General]
how to hide moving average priceLine ? after i searched in google, i found that i can hide candleSeries with below  code;
```
            candleSeries.applyOptions({
            priceLineVisible: false,
            lastValueVisible: false,
            });
```
but it not effect on moving average (SMA) priceLine. 

![hhhhhh](https://user-images.githubusercontent.com/37669470/179499035-5a199bf1-7a55-4f8e-b286-530f159e3b7e.png)

look , the both SMA priceLine are still visible. is there any option to hide them ?

**Comment by edew** [comment]
Answered here https://github.com/tradingview/lightweight-charts/issues/1125#issuecomment-1187122783
